profile/README.md

```diff
@@ -6,59 +6,52 @@
+| **Advanced Online Payments** |                                                  [Link](https://github.com/adyen-examples/adyen-java-spring-online-payments/tree/main/checkout-example-advanced)                                                  |                    [Link](https://github.com/adyen-examples/adyen-dotnet-online-payments/tree/main/checkout-example-advanced)                     |                                                 [Link](https://github.com/adyen-examples/adyen-node-online-payments/tree/main/checkout-example-advanced)                                                 | 
+|       **Recurring Payments** |                                                    [Link](https://github.com/adyen-examples/adyen-java-spring-online-payments/tree/main/subscription-example)                                                     |                       [Link](https://github.com/adyen-examples/adyen-dotnet-online-payments/tree/main/subscription-example)                       |                                                   [Link](https://github.com/adyen-examples/adyen-node-online-payments/tree/main/subscription-example)                                                    | 
+|               **Gift Cards** |                                                      [Link](https://github.com/adyen-examples/adyen-java-spring-online-payments/tree/main/giftcard-example)                                                       |                         [Link](https://github.com/adyen-examples/adyen-dotnet-online-payments/tree/main/giftcard-example)                         |                                                     [Link](https://github.com/adyen-examples/adyen-node-online-payments/tree/main/giftcard-example)                                                      |
+|              **Pay By Link** |                                                      [Link](https://github.com/adyen-examples/adyen-java-spring-online-payments/tree/main/paybylink-example)                                                      |                        [Link](https://github.com/adyen-examples/adyen-dotnet-online-payments/tree/main/paybylink-example)                         |                                                     [Link](https://github.com/adyen-examples/adyen-node-online-payments/tree/main/paybylink-example)                                                     |
+
+
+| Plugins                                                                    | 
```

Comment:
Great work! The headless integration example is not showcased here. Is it to have headless or advance integration option for plugins. The Salesforce team are also working on a headless sample.

Reply:
Plugins are also online payments only correct?

Reply:
Yes it is but it does not fit in the language/framework table. All the use case listed above can all be done with one plugin sample.

Reply:
I've turned it into: headless and plugin-integration (for now), let me know what you think @ayodejidev